tests/test_af3.py

```diff
@@ -37,6 +37,8 @@ def test_atom_ref_pos_to_atompair_inputs():
 
     atom_ref_pos_to_atompair_inputs(atom_ref_pos, atom_ref_space_uid)
 
+    assert atompair_inputs.shape == (16, 16, 5)
```

Comment:
this is already contracted by [this line](https://github.com/lucidrains/alphafold3-pytorch/blob/main/alphafold3_pytorch/alphafold3.py#L149)

Reply:
but we can still add it, you'll need to do an assignment of `atompair_inputs` above